rasa_core/policies/ensemble.py

```diff
@@ -55,6 +57,25 @@ def _training_events_from_trackers(training_trackers):
 
         return events_metadata
 
+    def _check_priorities(self) -> None:
+        """Checks for duplicate policy priorities within PolicyEnsemble."""
+
+        priority_dict = {}
```

Comment:
add import `from collections import defaultdict`

```suggestion
        priority_dict = defaultdict(list)
```